gui-apps/waybar/waybar-0.9.5.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="Highly customizable Wayland bar for Sway and Wlroots based compositors"
+HOMEPAGE="https://github.com/Alexays/Waybar"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/Alexays/${PN^}.git"
+else
+	SRC_URI="https://github.com/Alexays/${PN^}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+S="${WORKDIR}/${PN^}-${PV}"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="mpd network popups pulseaudio sndio tray +udev wifi"
+
+BDEPEND="
+	>=app-text/scdoc-1.9.2
+	virtual/pkgconfig
+"
+DEPEND="
+	dev-cpp/gtkmm:3.0
+	dev-libs/jsoncpp:=
+	dev-libs/libinput:=
+	dev-libs/libsigc++:2
+	>=dev-libs/libfmt-5.3.0:=
+	>=dev-libs/spdlog-1.8.0:=
+	dev-libs/date:=
+	dev-libs/wayland
+	dev-libs/wayland-protocols
+	gui-libs/wlroots
+	x11-libs/gtk+:3[wayland]
+	mpd? ( media-libs/libmpdclient )
+	network? ( dev-libs/libnl:3 )
+	popups? ( gui-libs/gtk-layer-shell )
+	pulseaudio? ( media-sound/pulseaudio )
+	sndio? ( >=media-sound/sndio-1.7 )
```

Comment:
this is currently not possible. `sndio` is not available in gentoo.
It lives in guru only. Maybe i was wrong with asking for this option _now_. Maybe it should be done in a later revision.
This would block your update for until this is somehow resolved (which is unclear it would be done).